src/services/proposals-deposits.service.ts

```diff
@@ -0,0 +1,50 @@
+import { Inject, Injectable } from '@nestjs/common';
+
+import { Repository } from 'typeorm';
+
+import { ProposalsDepositsEntity } from '@app/database';
+import { Coin } from '@lum-network/sdk-javascript/build/codec/cosmos/base/v1beta1/coin';
+
+@Injectable()
+export class ProposalsDepositsService {
+    constructor(@Inject('PROPOSALS_DEPOSITS_REPOSITORY') private readonly _repository: Repository<ProposalsDepositsEntity>) {}
+
+    getById = async (proposalId: number): Promise<ProposalsDepositsEntity> => {
```

Comment:
This should be named `getByProposalID` and we should have a `getById` method that takes a string (composite key)